src/views/new-asset-view/layout.tsx

```diff
@@ -16,16 +16,12 @@ export const NewPropertyLayout = (): JSX.Element => {
 
   const renderNewPropertyLink = () => {
     return (
-      // Only include link if visible on MMH
-      (newProperty?.assetType === "Dwelling" ||
-        newProperty?.assetType === "LettableNonDwelling") && (
-        <div>
-          {" "}
-          <Link as={RouterLink} to={`/property/${newProperty?.id}`}>
-            View property
-          </Link>{" "}
-        </div>
-      )
+      <div>
+        {" "}
+        <Link as={RouterLink} to={`/property/${newProperty?.id}`}>
+          View property
+        </Link>{" "}
+      </div>
     );
   };
 
```

Comment:
This logic is very simple. Can it be rendered in the component directly?

Reply:
Yes, I agree now that it's become simpler

Reply:
Done